deploy/olm-catalog/istio-workspace/manifests/istio-workspace.clusterserviceversion.yaml

```diff
@@ -0,0 +1,250 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "maistra.io/v1alpha1",
+          "kind": "Session",
+          "metadata": {
+            "name": "test-session"
+          },
+          "spec": {
+            "ref": [
+              {
+                "name": "ratings-v1"
+              }
+            ],
+            "route": {
+              "name": "end-user",
+              "type": "header",
+              "value": "jason"
+            }
+          }
+        }
+      ]
+    capabilities: Basic Install
+  name: istio-workspace.v0.0.3
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
```

Comment:
WARN[0000] Required csv fields not filled in file istio-workspace.clusterserviceversion.yaml:
	spec.description
	spec.provider

Reply:
Good point - it was wiped out in subsequent generation I believe :)

Reply:
c19d0d8